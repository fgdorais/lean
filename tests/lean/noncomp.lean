open nat

axiom n : nat

definition f (x : nat) :=  -- Error this is not computable
x + n

noncomputable definition f (x : nat) := x + n
