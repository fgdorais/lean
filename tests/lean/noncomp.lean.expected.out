noncomp.lean:5:0: error: definition 'f' is noncomputable, it depends on 'n'
